package-lock.json

```diff
@@ -0,0 +1,10103 @@
+{
```

Comment:
Since this is a yarn-based repo, `package-lock` should probably be removed. (I know I asked to add it, my bad!)